src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -11538,12 +11567,12 @@ class C3 : I1<C3>
                     throw new System.NotImplementedException();
                 }
 
-                public static explicit operator checked string(C3 x)
+                static explicit I1<C3>.operator checked string(C3 x)
```

Comment:
why did these change?

Reply:
This is an improvement, where the explicit interface implementation feature now actually explicitly implements operators like this one. All other static and non-static members were being explicitly implemented, whereas operators were not, despite the language allowing it.